models/maat/R/model2netcdf.MAAT.R

```diff
@@ -126,13 +131,20 @@ model2netcdf.MAAT <- function(rundir, outdir, sitelat = -999, sitelon = -999, st
                           seq(strptime(sub.maat.dates[1], format = "%Y-%m-%d", tz=timezone), by = "days", length = 1)))
       timestep.s <- day_secs / dims[1] # e.g. 1800 = 30 minute timesteps
       dayfrac <- 1 / dims[1]
-      day.steps <- seq(0, 0.99, 1 / dims[1])
+      day.steps <- head(seq(0, 1, by = dayfrac), -1)
```

Comment:
`tail` may be less confusing here.

```suggestion
      day.steps <- tail(seq(0, 1, by = dayfrac), 1)
```

Reply:
Not in this case as it needs to be DOY in XXX.XXX and going to one would mean we jump days, correct?

Reply:
in other words, I think I do want:

```
> day.steps
 [1] 0.00000000 0.04166667 0.08333333 0.12500000 0.16666667 0.20833333 0.25000000 0.29166667 0.33333333 0.37500000 0.41666667 0.45833333
[13] 0.50000000 0.54166667 0.58333333 0.62500000 0.66666667 0.70833333 0.75000000 0.79166667 0.83333333 0.87500000 0.91666667 0.95833333
```

and not

```
> day.steps
 [1] 0.04166667 0.08333333 0.12500000 0.16666667 0.20833333 0.25000000 0.29166667 0.33333333 0.37500000 0.41666667 0.45833333 0.50000000
[13] 0.54166667 0.58333333 0.62500000 0.66666667 0.70833333 0.75000000 0.79166667 0.83333333 0.87500000 0.91666667 0.95833333 1.00000000

```

Reply:
so we get say,

105.0, 105.042, 105.08, ...., 105.95833333, then the next day is 106.000

Reply:
because again we are starting at 0 not 1 and then we create time_bounds which provides the exact averaging period, like:

0, 0.042
0.042, 0.083
....

etc

Reply:
Ah, yes, you're right.